paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -271,41 +271,45 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         _collectBankAccountResult.tryEmit(result)
         when (result) {
             is CollectBankAccountResultInternal.Completed -> {
```

Comment:
we should not be using `CollectBankAccountResultInternal` here, but `CollectBankAccountResult` makes the intent not-nullable. I'll open a (breaking) different PR handling deferred intents in `CollectBankAccountLauncher`.